### PR TITLE
Ticket14 ineligible criteria page

### PR DESCRIPTION
**Pull request by RainbowGiantSquid**
backend @sergiomarrocoli 
I've added styles - all in the same branch. Best way to submit new form on /criteria/new to validate to ineligible.

### CODE REVIEW

**Comment by sergiomarrocoli** [comment]
front end looks good to me. only thing is on bigger screens it looks a bit stretch compared to designs
image

File: app/views/criteria/ineligible.html.erb [hunk 1]
```diff
@@ -1 +1,34 @@
...
+    </ul>
+
+    <div class="ineligible__links">
+      <div>
+        <p class="bold"><%= t('views.titles.action_agenda') %></p>
+        <p><%= t('views.criterium.ineligible.links.action_agenda.description') %></p>
+      </div>
+      <p>
+      <%= link_to('https://www.cbd.int/action-agenda/', title: 'Visit the Action Agenda website', target: '_blank') do %>
+        <span><%= t('views.buttons.access').upcase %></span>
```
Comment (yndajas) [inline app/views/criteria/ineligible.html.erb:26]: This should be 18px

Comment (yndajas) [inline app/views/criteria/ineligible.html.erb:26]: This should be 18px

File: app/views/criteria/ineligible.html.erb [hunk 2]
```diff
@@ -1 +1,34 @@
-Ineligible Criteria page
\ No newline at end of file
+<div class="ineligible">
+  <span class="ineligible__icon"></span>
+  <div class="container">
```
Comment (yndajas) [inline app/views/criteria/ineligible.html.erb:3]: The main colour here in the design is #343434 - I've created this in the dashboard branch as `$grey-xxdark` but happy to stick with the already defined #242424 colour variable if preferred

File: app/views/criteria/ineligible.html.erb [hunk 3]
```diff
@@ -1 +1,34 @@
...
+    <h1><%= t('views.criterium.ineligible.title') %></h1>
+
+    <p>
+      <%= t('views.criterium.ineligible.description') %>
+    </p>
+    <p class="bold">
+      <%= t('views.criterium.ineligible.results_did_not_pass') %>
+    </p>
```
Comment (yndajas) [inline app/views/criteria/ineligible.html.erb:11]: Probably fine but the spacing in the design is a bit more top-heavy

File: app/assets/stylesheets/components/_ineligible.scss [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  }
+
+  h1 {
+    font-family: $header-font;
+  }
+
+  p,
+  li {
+    line-height: $p-font-line-height;
+    font-size: rem-calc(20);
```
Comment (yndajas) [inline app/assets/stylesheets/components/_ineligible.scss:33]: Property ordering

**Comment by RainbowGiantSquid** [comment]
> front end looks good to me. only thing is on bigger screens it looks a bit stretch compared to designs image

ok, thanks. I'll adjust the flex - should do the job

**Review by yndajas — CHANGES_REQUESTED** [review]
Looks good! Just a few small details on the styles

File: app/views/criteria/ineligible.html.erb (hunk 2 above)
Comment (RainbowGiantSquid) [reply app/views/criteria/ineligible.html.erb:3]: Yeah I know, I was oing to leave it as is to avoid "50 shades of grey", if that's ok.

**Review by yndajas — APPROVED** [review]

**Pull request merged**